src/material-experimental/mdc-slider/slider.ts

```diff
@@ -331,6 +350,46 @@ export class MatSliderThumb implements AfterViewInit {
     }
   }
 
+  _onBlur(): void {
+    this._onTouched();
+    this._blur.emit();
+  }
+
+  /**
+   * Sets the model value. Implemented as part of ControlValueAccessor.
+   * @param value
+   */
+  writeValue(value: any): void {
+    this.value = value;
+  }
+
+  /**
+   * Registers a callback to be triggered when the value has changed.
+   * Implemented as part of ControlValueAccessor.
+   * @param fn Callback to be registered.
+   */
+  registerOnChange(fn: any): void {
+    this._onChange = fn;
+  }
+
+  /**
+   * Registers a callback to be triggered when the component is touched.
+   * Implemented as part of ControlValueAccessor.
+   * @param fn Callback to be registered.
+   */
+  registerOnTouched(fn: any): void {
+    this._onTouched = fn;
+  }
+
+  /**
+   * Sets whether the component should be disabled.
+   * Implemented as part of ControlValueAccessor.
+   * @param isDisabled
+   */
+  setDisabledState(isDisabled: boolean): void {
+    this._slider.disabled = isDisabled;
```

Comment:
I'm not super familiar with the setup, but wouldn't this disable the entire slider when one of the thumbs is disabled? I was under the impression that they can be disabled independently.

Reply:
No, unfortunately that's not possible with the way the mdc-slider foundation was implemented

Reply:
But to answer your other question, yes this would disable the entire slider if one of the thumbs was disabled

Reply:
It's not a blocker, but wouldn't this cause the values of the two thumbs to compete with each other? E.g. if the first one gets called with `setDisabledState(true)` and the second one is `setDisabledState(false)`, the slider wouldn't be disabled.

Reply:
I think one way to handle this would be:
```ts
this._disabled = isDisabled;
this._slider._updateDisabled();
```

and `_updateDisabled` would look like this:
```ts
_updateDisabled() {
  this.disabled = this._inputs.some(input => input._disabled)
}
```

Reply:
@crisbeto Yes, they would be competing with each other
@mmalerba Just to make sure we're on the same page here, that would make it so that if either slider thumb is disabled the entire slider thumb is disabled? In other words, kristiyan's example of calling `setDisabledState(true)` on one thumb and `setDisabledState(false)` on the other would result in a disabled slider